docker-compose.yml

```diff
@@ -21,6 +21,7 @@ backend:
     VOLONTULO_SECRET_KEY: a63eb5ef-3b25-4595-846a-5d97d99486f0
     VOLONTULO_DB_HOST: db
     VOLONTULO_DB_PASS: volontulo
+    PYTHONUNBUFFERED: 1
```

Comment:
what's the reason of puting this variable here?

Reply:
@jacol12345, in fact I don't remember why we put it here?

Reply:
I think it can stay, you see some example of the issue it solves eg. there https://mail.python.org/pipermail/tutor/2003-November/026645.html